### PR TITLE
update samples to use java-chassis 1.3.0

### PR DESCRIPTION
**Pull request by liubao68**

### CODE REVIEW

File: java-chassis-samples/pom.xml [hunk 1]
```diff
@@ -25,6 +25,14 @@
   <name>Java Chassis::Samples</name>
   <packaging>pom</packaging>
   <version>2.0.0-SNAPSHOT</version>
```
Comment (WillemJiang) [inline java-chassis-samples/pom.xml]: We may also update the version here.

Comment (liubao68) [reply java-chassis-samples/pom.xml]: updated

**Comment by WillemJiang** [comment]
@liubao68  We may consider to do a release about this example.

**Comment by liubao68** [comment]
> @liubao68 We may consider to do a release about this example.

@WillemJiang I thought about this. servicecomb-samples now contain many projects, it's make a bit harder to do a release. I think we only keep the lastest examples is a good idea, and I am not preparing release now. We may think a better way to manage examples.

**Pull request merged**